eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -10,7 +10,7 @@ parameters:
   TestName: LiveTest
   TimeoutInMinutes: 60
   TestStepMavenInputs:
-    options: '--batch-mode --fail-at-end -Dmaven.wagon.http.pool=false -Dsurefire.rerunFailingTestsCount=3 --settings eng/settings.xml'
+    options: '--batch-mode -Dmaven.wagon.http.pool=false -Dsurefire.rerunFailingTestsCount=3 --settings eng/settings.xml'
```

Comment:
We definitely want to have this using `fail-at-end` given that we have many multi module services and this would result in latter modules never reporting tests until earlier ones pass.

Reply:
Yes, will revert it back.